blog/2022/12-7-first-look-at-resoto-dashboards/index.md

```diff
@@ -0,0 +1,76 @@
+---
+authors: [pablo]
+tags: [ui, dashboard, visualization]
+---
+
+# First look at resoto dashboards
+
+Resoto makes it easier than ever to create an inventory of your cloud infrastructure. And with just a few commands, you can immediately get a snapshot of what is currently happening with your resources. Furthermore, with this data ingested into a time series database, you not only know the current state of your cloud but also how they evolve.
+
+So, with all this data now available, you will probably want a simple and human-readable way to visualize it. Until now, you could [export the data to google sheets](https://resoto.com/docs/how-to-guides/data-export/export-data-to-google-sheets) to create some charts out of it or use [resoto notebook](https://resoto.com/docs/reference/notebook) to interactively visualize and analyze the result of your searches.
+
+Now, we are introducing a new way to have this visual information always available: Resoto Dashboards.
+
+You may be wondering: "But why do we need yet another way to visualize data?". With our dashboards, you don't just visualize the data. Your searches and queries will be executed periodically and the results displayed in widgets automatically, so you always look at updated information. You can organize your widgets freely in the dashboard, to keep the most important metrics always within reach. You can share these dashboards across your organization so everybody that needs this information can make use of it. And of course, you can access your dashboards from everywhere as long as you have access to your resoto installation.
+
+## Key concepts
+
+A _dashboard_ is a container where you can place different _widgets_ to show relevant information. Each _widget_ has its way to display data, and thus, its configurable properties. Available widgets so far are:
+
+- Numeric Indicators
+- Line Charts
+- Heatmaps
+- Tables
+
+Of course, you can freely position and resize widgets inside a dashboard, to organize the data in the best way possible for your application.
+
+Each widget needs one or more _data sources_ to get the data that the user wants to display. These data sources have a query that is executed periodically to keep the dashboard updated. Each widget may have some limitations on what data sources it can accept, so not all data sources are available for every widget. Available data sources so far are:
+
+- Search
+- Aggregate Search
+- Time Series
+- Two-entry Aggregate Search
+
+## Resoto Example Dashboard
+
+Once you have resoto up and running, you can access the UI from a web browser in `HTTP://<your_resoto_url>/ui`. The following home screen should appear:
```

Comment:
should appear, after entering your PSK. I think there needs to be a short sentence about how to acquired the PSK.

Reply:
We should link here to the `Launching the Web UI` sections, that we added to each section in https://resoto.com/docs/getting-started/install-resoto

Reply:
If we link to those sections, do we still need to mention the PSK? Because that should be covered there right?

Reply:
It's mentioned explicitly in the AWS setup guide, not in the others. But that may be because they work differently. Not an expert 😬 @lloesche ?